client/src/components/App.jsx

```diff
@@ -40,11 +56,48 @@ class App extends React.Component {
     )
   }
 
-  render(){
+  renderFilteredData(event) {
+    event.preventDefault();
+    this.setState({
+      selectMenu: {value: event.target.value}
+    })
+  };
+
+  filterDataNewestHelperFunction(list) {
+    list.forEach((review) => {
+      review.time = new Date(review.review_time);
+    });
+    return list.sort((a,b) => {
+      return b.time-a.time;
+    });
+  };
+
+  sortHighestRatingsFirst(list) {
+    return list.sort((a,b) => {
+      return b.overall_rating - a.overall_rating;
+    });
+  };
+ 
+  sortLowestRatingsFirst(list) {
+    return list.sort((a,b) => {
+      return a.overall_rating - b.overall_rating;
+    });
+  };
+
+  render(){ 
     return (
       <div>
-       
-        {this.state.reviews.length > 0 ? this.renderReviews() : this.renderNoReviews()}
+        <div>
+          <Header reviewsCount={this.state.reviews.length} restaurantInfo = {this.state.reviews}/>
```

Comment:
`restaurantInfo = {this.state.reviews}` -> `restaurantInfo={this.state.reviews}`